scripts/actions/spells/trust/elivira.lua

```diff
@@ -13,6 +13,26 @@ end
 
 spellObject.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
+	
+		mob:addSimpleGambit(ai.t.SELF, ai.c.NOT_STATUS, xi.effect.BARRAGE,
+                        ai.r.JA, ai.s.SPECIFIC, xi.ja.BARRAGE)
+						
+		mob:addSimpleGambit(ai.t.SELF, ai.c.NOT_STATUS, xi.effect.BERSERK,
+                        ai.r.JA, ai.s.SPECIFIC, xi.ja.BERSERK)
+	
+		mob:addSimpleGambit(ai.t.SELF, ai.c.NOT_STATUS, xi.effect.DOUBLE_SHOT,
+                        ai.r.JA, ai.s.SPECIFIC, xi.ja.DOUBLE_SHOT)
+	
+	mob:addSimpleGambit(ai.t.SELF, ai.c.NOT_STATUS, xi.effect.DECOY_SHOT,
+                        ai.r.JA, ai.s.SPECIFIC, xi.ja.DECOY_SHOT)
+
+	mob:addSimpleGambit(ai.t.TARGET, ai.c.ALWAYS, 0, ai.r.RATTACK, 0, 0)
+
+    mob:setAutoAttackEnabled(true)
+	
+	mob:setMobMod(xi.mobMod.TRUST_DISTANCE, xi.trust.movementType.NO_MOVE)
```

Comment:
So I'm reading this right: She'll spawn, she'll spam RA, she won't move, and if a mob gets close to her she won't reposition and she'll start meleeing? 

What happens on retail? I've never used her I don't think